CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # CHANGELOG
 
+## 2.10.1 (2024-02-06)
+
+MLflow 2.10.1 is a patch release, containing fixes for various bugs in the `transformers` and `langchain` flavors, the MLflow UI, and the S3 artifact store. More details can be found in the patch notes below.
+
+Bug fixes:
```

Comment:
Should we list #10952 here as well? We mention that this version fixes a bug in Langchain above, so it might be helpful for users to tell what was it.

Reply:
done! added the artifact store fix to the top-level notes too